core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -1313,6 +1313,15 @@ export class LoggedOutUser extends BaseUser {
       ? readBlogPostMobileButtonInPartnershipsPage
       : readBlogPostDesktopButtonInPartnershipsPage;
 
+    if (this.isViewportAtMobileWidth()) {
+      const matCardContentSelector = '[data-test="mat-card-content"]';
```

Comment:
Store it as a constant.

Reply:
Also, please put a comment to explain this part.

Reply:
Stored as a constant and made a comment to elaborate the action. 

Thanks.